air/src/air/context.rs

```diff
@@ -124,7 +124,7 @@ impl<B: StarkField> AirContext<B> {
 
         // validate Lagrange kernel aux column, if any
         if let Some(lagrange_kernel_aux_column_idx) = lagrange_kernel_aux_column_idx {
-            assert!(lagrange_kernel_aux_column_idx < trace_info.get_aux_segment_width(), "Lagrange kernel column index out of bounds: index={}, but only {} columns in segment", lagrange_kernel_aux_column_idx, trace_info.get_aux_segment_width());
+            assert!(lagrange_kernel_aux_column_idx == trace_info.get_aux_segment_width() - 1, "Lagrange kernel column should be the last column of the auxiliary trace: index={}, but aux trace width is {}", lagrange_kernel_aux_column_idx, trace_info.get_aux_segment_width());
```

Comment:
minor nit: could we fix the formatting of this line (it was odd before this PR, I'm wondering why `rustfmt` didn't catch it).

Reply:
I am not sure why. I have fixed it now